src/server/pfs/server/driver.go

```diff
@@ -1867,6 +1869,7 @@ func (d *driver) putFile(ctx context.Context, file *pfs.File, delimiter pfs.Deli
 		return err
 	}
 
+	limiter := limit.New(putObjectConcurrency)
```

Comment:
It's not a particularly big deal but why did you move this up here? I generally like to keep declarations close to where they're used.

Reply:
I put it there to raise its scope -- we use the limiter below when putting the objects for the header/footer ... around line 2012. I thought about omitting the use of a limiter there altogether since its just 2 additional puts per putfile ... but figured it was easy enough to use the limiter that was already there.

Reply:
Generally speaking raising a variables scope isn't a great thing to do unless you it's necessary. This is really all the same scope, just higher up in the scope. Still I think it's worth moving it down so it's declared close to where it's used.

Reply:
It's not the same scope. If I leave the limiter declaration as-is it generates a compiler error. So you want me to declare two limiters? One for the put objects for the content and the other just for the header/footer?